src/libs/actions/Report.js

```diff
@@ -226,6 +265,14 @@ function getSimplifiedReportObject(report) {
         visibility,
         isOwnPolicyExpenseChat: lodashGet(report, ['isOwnPolicyExpenseChat'], false),
     };
+
+    const currentMaxSeqNumber = lodashGet(allReports, [report.reportID, 'maxSequenceNumber'], 0);
+
+    // Current local data is newer than report data from fetchReport. For example if Pusher new comment event is comming while API request of fetchReport isn't completed.
```

Comment:
This is not a good example of a comment. It does not explain why and why is happening in the next line. Please try to be more specific. 

Why did we remove the optimistic action count?